butterknife/build.gradle

```diff
@@ -6,6 +6,7 @@ android {
 
   defaultConfig {
     minSdkVersion rootProject.ext.minSdkVersion
+    targetSdkVersion rootProject.ext.targetSdkVersion
```

Comment:
Libraries should not have target versions. This can cause problems with manifest merging.